pallets/multisig/src/lib.rs

```diff
@@ -203,6 +210,28 @@ decl_module! {
 
         fn deposit_event() = default;
 
+        fn on_runtime_upgrade() -> Weight {
+            use sp_version::RuntimeVersion;
+
+            fn kill(item: &[u8]) {
+                use frame_support::{StorageHasher, Twox128};
+                let mut prefix = Vec::new();
+                prefix.extend_from_slice(&Twox128::hash(b"multisig"));
+                prefix.extend_from_slice(&Twox128::hash(item));
+                kill_prefix(&prefix)
+            }
+
+            let current_version = <T::Version as Get<RuntimeVersion>>::get().transaction_version;
+            if current_version < TransactionVersion::get() {
+                TransactionVersion::set(current_version);
+                kill(b"proposals")
+                kill(b"proposal_detail")
+                kill(b"proposal_ids")
+            }
+
+            1_000
```

Comment:
In this case I think we need a reasonable value since this is not a migration.

Reply:
Let's add a todo for now and get back to this later when we benchmark this pallet.